superset/config.py

```diff
@@ -950,7 +950,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # indefinite.
 SIP_15_ENABLED = True
 SIP_15_GRACE_PERIOD_END: Optional[date] = None  # exclusive
-SIP_15_DEFAULT_TIME_RANGE_ENDPOINTS = ["unknown", "inclusive"]
+SIP_15_DEFAULT_TIME_RANGE_ENDPOINTS = ["inclusive", "exclusive"]
```

Comment:
I'm surprised these are not in line with the comment in `utils`. I would be interested in hearing @john-bodley's thoughts on this change, as it looks like this is a bug.